examples/meta/generator/targets/cpp.json

```diff
@@ -11,7 +11,7 @@
     "Comment": "//$comment\n",
     "Init": {
         "Construct": "auto $name = some<C$typeName>($arguments)$kwargs",
-        "Copy": "auto $name = $expr$kwargs",
+        "Copy": "auto $name = wrap($expr)$kwargs",
```

Comment:
@lisitsyn re-introduced wrap since we will get memory errors otherwise (things are not ref'ed and then deleted by objects)

Reply:
This is a bit strange. Copy ctor doesn't ref?

Reply:
The point here is if you do
'labels = svm->precict'
then Shogun returns a raw pointer. Now if you pass this to another object that increases the ref-counter, and then decreases it, the object is destroyed. So calling a method on it subsequently crashes. This is why I re-added it, just like when you introduced it in the first place.
Or you mean something else?

Reply:
Yes I've got it now. I guess we change it once we get rid of raw pointers.